gdnative-bindings/build.rs

```diff
@@ -2,54 +2,41 @@ use gdnative_bindings_generator::*;
 
 use std::env;
 use std::fs::File;
-use std::io::BufWriter;
+use std::io::{BufWriter, Write as _};
 use std::path::PathBuf;
 use std::process::Command;
 
 fn main() {
     let out_path = PathBuf::from(env::var("OUT_DIR").unwrap());
 
-    let bindings_types_rs = out_path.join("bindings_types.rs");
-    let bindings_traits_rs = out_path.join("bindings_traits.rs");
-    let bindings_methods_rs = out_path.join("bindings_methods.rs");
+    let output_rs = out_path.join("generated.rs");
```

Comment:
Since there is only `generated.rs` now, you should only `include!` that file instead of the original three. This is causing the build to fail.

Reply:
Thank you! It had been using the old files locally and I missed it.